hardware/raspberrypi/boot_diagnostics.md

```diff
@@ -24,6 +24,6 @@ The diagnostic information is as follows:
 | tftp       | Network boot: TFTP server IP address|
 
 
-This display can be disabled using the DISABLE_HDMI option, see [Pi4 Bootloader Configuration](./bcm2711_bootloader_config.md).
+This display can be disabled using the `DISABLE_HDMI` option, see [Bootloader Configuration](./bcm2711_bootloader_config.md).
```

Comment:
Perhaps it's worth adding something like "(or delayed with the `HDMI_DELAY` option)" ? :shrug:

Reply:
I think anyone who cares about that will find it in the config page